include/pqxx/stream_to.hxx

```diff
@@ -102,9 +102,10 @@ public:
    *     the stream will write all columns in the table, in schema order.
    */
   static stream_to raw_table(
-    transaction_base &tx, std::string_view path, std::string_view columns = "")
+    transaction_base &tx, std::string_view path, std::string_view columns = "",
+    bool binary = false)
```

Comment:
I would recommend against using `bool` arguments for such things.  They render mistakes easy to make and hard to spot.  Use the `pqxx::format` enum instead.

Since that's a `class enum`, it ensures type-safety of the parameter.  On top of that, it gives us more extensibility in the future — adding an enum vaue will have very limited consequences for backward compatibility.